extra/mariabackup/xtrabackup.cc

```diff
@@ -4865,7 +4865,7 @@ static bool backup_wait_for_commit_lsn()
   ut_ad(metadata_to_lsn);
   metadata_last_lsn= lsn;
 
-  last_lsn= backup_wait_for_lsn_low(LSN_MAX);
+  last_lsn= backup_wait_for_lsn_low(lsn);
```

Comment:
This code had last been modified in #3370. I see that the logic was to keep polling the log, until the parsed log sequence number will not change within a 5-second interval. This change makes sense; it should avoid an extra 5-second wait and some unnecessary log copying.